client/src/pages/account/index.js

```diff
@@ -1,32 +1,19 @@
-import api from 'api'
+import React from 'react'
 import { Loader } from 'components/Loader'
 import { EnterEmailModal } from 'components/modals/EnterEmailModal'
 import { useSignIn } from 'hooks/useSignIn'
-import { useRouter } from 'next/router'
-import React from 'react'
+// import { useRouter } from 'next/router'
 
-export const Account = ({ noCode, orcid, accessToken, refreshToken }) => {
-  const router = useRouter()
-  const {
-    needsEmail,
-    orcidInfo,
-    setOrcidInfo,
-    setEmail,
-    setNeedsEmail,
-    setError,
-    user
-  } = useSignIn()
-
-  // If the user navigates to /account manually, redirect them to /account/basic-information
-  if (noCode && !needsEmail && user) {
-    router.replace('/account/basic-information')
-  }
+export const Account = ({ code, originUrl }) => {
+  // const router = useRouter()
```

Comment:
What's this about?

Reply:
got catch, the hook handles the redirect now so this can be removed will do!